tests/installation/installation_overview.pm

```diff
@@ -69,6 +69,20 @@ sub run {
         $self->deal_with_dependency_issues;
         assert_screen "inst-xen-pattern" if get_var('XEN');
         ensure_ssh_unblocked;
+        # Check the systemd target, see poo#45020
+        if (get_var('DESKTOP')) {
+            my $target;
```

Comment:
In case you like ternaries:
`my $target = check_var('DESKTOP', 'textmode') ?  "multi-user" : "graphical";`

Reply:
Well, that's shorter indeed. I'll do that.